packages/cli/src/templates/app-template.js

```diff
@@ -17,6 +20,37 @@ import './list';
 connectRouter(store);
 
 class AppComponent extends LitElement {
+
+  async connectedCallback() {
+    super.connectedCallback();
+    const route = window.location.pathname;
+    const response = await Promise.all([
+      await client.query({
+        query: ConfigQuery
+      }),
```

Comment:
Is it easier to instead include this data as part of the graph, from within the graph lifecycle, then queried as part of the graph query? Just spitballing here.

Reply:
Yeah, two queries here isn't ideal, that said, I think `config` should stay it's own schema / structure.  Mixing it up with the graph may not be super intuitive / maintainable.  

However, it would be great if **apollo-client** could support sending multiple queries for us, then we wouldn't have to use `Promise.all` but I suppose it could just be a limitation.  Would be good to think of another more elegant way if we can think of one.